src/Utils.re

```diff
@@ -32,3 +35,9 @@ let useClickOutside = (onClickOutside: Dom.mouseEvent => unit) => {
 
   elementRef;
 };
+
+/* Utils.Errors */
```

Comment:
Why to comment with the location of it?

Reply:
I separated the utils because they were looking too messy. I couldn't spot right away where each one started and ended, then I added these comments.

Reply:
Do you think that it could be done in a better way?

Reply:
aah I got it, I use the modules themselves as separators but nothing against this

Reply:
As some are just variables loose around, I had to use something else. Can I somehow use modules to separate each util?